Formula/s/swift-section.rb

```diff
@@ -0,0 +1,44 @@
+class SwiftSection < Formula
+  desc "CLI tool for parsing mach-o files to obtain Swift information"
+  homepage "https://github.com/MxIris-Reverse-Engineering/MachOSwiftSection"
+  url "https://github.com/MxIris-Reverse-Engineering/MachOSwiftSection/archive/refs/tags/0.4.1.tar.gz"
+  sha256 "afe48172e28e7ee626edf051d58662b62b63ddb6780e830cafbe886007af8029"
+  license "MIT"
+  head "https://github.com/MxIris-Reverse-Engineering/MachOSwiftSection.git",
+       branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  # The Package.swift file requires Swift 5.10 or later.
+  # But it is actually only builable with Swift 6.1+ due to the usage of trailing commma in comma-separated lists.
+  depends_on xcode: ["16.3", :build]
+  depends_on :macos
+
+  uses_from_macos "swift" => :build
+
+  def install
+    system "swift", "build", "--disable-sandbox", "-c", "release"
+    bin.install ".build/release/swift-section"
+  end
+
+  test do
+    (testpath/"test.swift").write <<~SWIFT
+      public struct MyTestStruct {
+          public let id: Int
+          public let name: String
+          public init(id: Int, name: String) {
+              self.id = id
+              self.name = name
+          }
+      }
+    SWIFT
+    system "swiftc", "-emit-library", "-module-name", "Test", "Test.swift", "-o", "libTest.dylib"
+    shell_output("#{bin}/swift-section dump libTest.dylib -o ./output.txt -s types")
```

Comment:
should be system command

Reply:
Could you explain more detailed on this? I'm not sure the context here.

Do you mean `system "#{bin}/swift-section" "dump" "libTest.dylib" "-o" "./output.txt" "-s" "types"`